velox/functions/lib/string/StringImpl.h

```diff
@@ -653,4 +653,84 @@ FOLLY_ALWAYS_INLINE void pad(
       padPrefixByteLength);
 }
 
+FOLLY_ALWAYS_INLINE static void
+initCapAscii(char* output, const char* input, size_t length) {
+  bool newWord = true;
+  for (size_t i = 0; i < length; ++i) {
+    if (std::isspace(input[i])) {
+      output[i] = input[i];
+      newWord = true;
+    } else {
+      output[i] = newWord ? std::toupper(input[i]) : std::tolower(input[i]);
+      newWord = false;
+    }
+  }
+}
+
+FOLLY_ALWAYS_INLINE size_t initCapUnicode(
```

Comment:
This is a long function. Why do you want to inline it? Same question about the next function.

Reply:
Comes down to inexperience of writing C++ I'm afraid, I was pattern matching from the other functions in the file